index.js

```diff
@@ -0,0 +1,16 @@
+let names=[]
+function myFunction()
+{
+ let x=document.getElementById("box");
```

Comment:
Is there we are using `let` instead of `const`?

Reply:
I understand it's a variable with a fixed value.....I'll refactor the code and create a PR